### ISSUE TITLE
OrleansHubLifetimeManager failed to start

### ISSUE THREAD

**Issue by michaelmcneilnet** [OrleansContrib/SignalR.Orleans#147]
Using the sample code from the readme and gets errors on starting the silo

```cs
using Microsoft.AspNetCore.SignalR;

// Create a host that can cohost aspnetcore AND orleans together in a single process.
var builder = WebApplication.CreateBuilder(args);
builder.Host.UseOrleans(siloBuilder =>
{
    siloBuilder.UseLocalhostClustering();
    siloBuilder.UseSignalR(); // Adds ability #1 and #2 to Orleans.
    siloBuilder.RegisterHub<MyHub>(); // Required for each hub type if the backplane ability #1 is being used.
});

builder.Services
    .AddSignalR()  // Adds SignalR hubs to the web application
    .AddOrleans(); // Tells the SignalR hubs in the web application to use Orleans as a backplane (ability #1)

var app = builder.Build();
app.MapHub<MyHub>("/myhub");
await app.RunAsync();

// A SignalR Hub. https://learn.microsoft.com/en-us/aspnet/core/signalr/hubs?view=aspnetcore-7.0
class MyHub : Hub
{
}
```

```
fail: Orleans.Runtime.SiloLifecycleSubject[100450]
      OrleansHubLifetimeManager failed to start due to errors at stage Active (20000)
      System.InvalidOperationException: Cannot resolve scoped service 'Orleans.Streams.StreamDirectory' from root provider.
         at Microsoft.Extensions.DependencyInjection.ServiceLookup.CallSiteValidator.ValidateResolution(Type serviceType, IServiceScope scope, IServiceScope rootScope)
         at Microsoft.Extensions.DependencyInjection.ServiceProvider.GetService(Type serviceType, ServiceProviderEngineScope serviceProviderEngineScope)
         at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
         at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
         at Orleans.Runtime.Providers.SiloStreamProviderRuntime.GetStreamDirectory() in /_/src/Orleans.Streaming/Providers/SiloStreamProviderRuntime.cs:line 127
         at Orleans.Providers.Streams.Common.PersistentStreamProvider.GetStream[T](StreamId streamId) in /_/src/Orleans.Streaming/PersistentStreams/PersistentStreamProvider.cs:line 158
         at Orleans.StreamExtensions.GetServerStream(IStreamProvider streamProvider, Guid serverId)
         at SignalR.Orleans.OrleansHubLifetimeManager`1.EnsureStreamSetup()
         at Orleans.Runtime.SiloLifecycleSubject.MonitoredObserver.OnStart(CancellationToken ct) in /_/src/Orleans.Runtime/Lifecycle/SiloLifecycleSubject.cs:line 134
```

**Comment by galvesribeiro** [comment]
Thanks for the report. I've identified where it is throwing that. Will fix on 7.0.1.

**Comment by galvesribeiro** [comment]
This was fixed on #148 and the package released https://www.nuget.org/packages/SignalR.Orleans/7.0.1

Thanks for the report!
